banners/english/WMDE_FR_2025_Desktop_EN_01/content/BannerSlidesVar.vue

```diff
@@ -2,27 +2,28 @@
 	<KeenSliderSlide :is-current="currentSlide === 0">
 		<p class="headline">
 			<InfoIcon fill="#990a00"/>
-			{{ liveDateAndTime.currentDate }}, {{ liveDateAndTime.currentTime }}: &#8220;Wikipedia still can't be
-			sold.&#8221; - An important update for readers in Germany.
+			{{ liveDateAndTime.currentDate }}, {{ liveDateAndTime.currentTime }} - “Wikipedia still can't be sold.” -
```

Comment:
I'm not sure what the reason was to remove `&#8220;` here and use “ instead, but maybe not that important..